tagging/utils.py

```diff
@@ -3,12 +3,25 @@
 calculation.
 """
 import math
-import types
 
 from django.db.models.query import QuerySet
-from django.utils.encoding import force_unicode
+try:
+    from django.utils.encoding import force_unicode
+except ImportError:
+    def force_unicode(s, *args, **kwargs):
+        return s
```

Comment:
FWIW, django-compat does this:  https://github.com/arteria/django-compat/blob/f55d127ec84ef165b0a8567805220031c373c599/compat/__init__.py#L61